pallets/services/src/mock.rs

```diff
@@ -479,28 +445,28 @@ pub fn new_test_ext_raw_authorities(authorities: Vec<(AccountId)>) -> sp_io::Tes
 		},
 	);
 
-	let cggmp21_job_result_verifier_json: serde_json::Value = serde_json::from_str(include_str!(
-		"../../../forge/out/CGGMP21Hooks.sol/CGGMP21JobResultVerifier.json"
-	))
-	.unwrap();
-
-	let cggmp21_job_result_verifier_code = hex::decode(
-		cggmp21_job_result_verifier_json["deployedBytecode"]["object"]
-			.as_str()
-			.unwrap()
-			.replace("0x", ""),
-	)
-	.unwrap();
-
-	evm_accounts.insert(
-		CGGMP21_JOB_RESULT_VERIFIER,
-		fp_evm::GenesisAccount {
-			code: cggmp21_job_result_verifier_code,
-			storage: Default::default(),
-			nonce: Default::default(),
-			balance: Default::default(),
-		},
-	);
+	// let cggmp21_job_result_verifier_json: serde_json::Value = serde_json::from_str(include_str!(
```

Comment:
@shekohex for reviews to be ready to merge we shouldn't leave comments and things in here.

Reply:
what do you suggest should be done here? the contract code that generates that part is commented out (https://github.com/webb-tools/tangle/blame/11ce26d70589719324087cfee9cf21971046833a/forge/src/cggmp21/CGGMP21Hooks.sol#L111) this causes the CI to fail. Should I delete all commented out code?

Reply:
I think we should probably delete things we comment out like this. Let's merge in any case now.